sirepo/template/openmc.py

```diff
@@ -46,7 +46,8 @@ def background_percent_complete(report, run_dir, is_running):
 def get_data_file(run_dir, model, frame, options):
     sim_in = simulation_db.read_json(run_dir.join(template_common.INPUT_BASE_NAME))
     if model == "geometry3DReport":
-        return _SIM_DATA.dagmc_filename(sim_in)
+        d, s = _SIM_DATA.dagmc_and_maybe_step_filename(sim_in)
+        return s or d
```

Comment:
In this case, I think it should always download the h5m file because the panel that provides the download is "DAGMC Geometry".